premium/usage.go

```diff
@@ -99,7 +102,9 @@ func WithMinTimeBetweenFlushes(minTimeBetweenFlushes time.Duration) UsageClientO
 // WithMaxRetries sets the maximum number of retries to update the usage in case of an API error
 func WithMaxRetries(maxRetries int) UsageClientOptions {
 	return func(updater *BatchUpdater) {
-		updater.maxRetries = maxRetries
+		if maxRetries > 0 {
```

Comment:
This means that retries can never be turned off?

Reply:
maxRetries = 1 would be "single request, no retries" but I didn't want to be pedantic about the loop and the descriptions.